vllm/v1/core/scheduler_output.py

```diff
@@ -24,6 +27,8 @@ class NewRequestData:
     block_ids: List[int]
     num_computed_tokens: int
     lora_request: Optional["LoRARequest"]
+    grammar: Optional["Grammar"]
+    bitmask: Optional["torch.Tensor"]
```

Comment:
nit: bitmask is too ambiguous, should be something like `logits_bitmask` or `grammar_bitmask`

Reply:
I would make a GrammarRequest to put all related grammar stuff in there for future proof.